src/components/ebay-date-textbox/component.ts

```diff
@@ -79,6 +79,7 @@ class DateTextbox extends Marko.Component<Input, State> {
 
     calculateNumMonths() {
         this.state.numMonths =
+            typeof document !== "undefined" &&
```

Comment:
I think it's fine,  but generally we put these calls into the lifecycle methods. So lets move this check to `onCreate` and then only call calulateNumMonths if that's defined.

Reply:
Good call, will move.